service/main/urls.py

```diff
@@ -14,16 +14,17 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from rest_framework.urlpatterns import format_suffix_patterns
+from django.urls import path, include
+# from rest_framework.urlpatterns import format_suffix_patterns
 
 from api import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('movies/', views.get_movies),
+    path('api/', include('api.urls')),
```

Comment:
According to spec this prefix should be `/api/v1`:
https://github.com/WaveOfCoding/watchtower/blob/main/service/spec/http/api.yaml#L7